Feature: Show the name of the NewGRF in the build vehicle window. (#7852)
OpenTTD/OpenTTD: src/lang/english.txt

STR_CONFIG_SETTING_POPULATION_IN_LABEL_HELPTEXT                 :Display the population of towns in their label on the map
STR_CONFIG_SETTING_GRAPH_LINE_THICKNESS                         :Thickness of lines in graphs: {STRING2}
STR_CONFIG_SETTING_GRAPH_LINE_THICKNESS_HELPTEXT                :Width of the line in the graphs. A thin line is more precisely readable, a thicker line is easier to see and colours are easier to distinguish
STR_CONFIG_SETTING_SHOW_NEWGRF_NAME                             :Show the NewGRF's name in the build vehicle window

> STR_CONFIG_SETTING_SHOW_NEWGRF_NAME_HELPTEXT                    :When enabled the name of the NewGRF which the selected vehicle comes from displayed at the bottom of the build vehicle window

Reviewer: ```suggestion
STR_CONFIG_SETTING_SHOW_NEWGRF_NAME_HELPTEXT                    :Add a line to the build vehicle window, showing which NewGRF the selected vehicle comes from.
```
Or at least add a comma after "When enabled", and a missing word in "is displayed" :)

Author: I accept your suggestion, that sounds better. :)
